pyproject.toml

```diff
@@ -17,7 +17,7 @@ classifiers = [
 dependencies = [
     "crds",
     "colorama>=0.4.1",
-    "pytest>=6",
+    "pytest>=9",
```

Comment:
pytest 9 was only release 2025-11-08 (less than a month ago at the time of writing this). Too soon?